File: palindrome.js
var isPalindrome = function(x) {
    if (x < 0){
        return false;
    }
    let str = x.toString();
    let s = 0;
    let e = str.length - 1;
    while(s < e){
        if (str.charAt(s++) != str.charAt(e--)){
            return false;
        }
    }
    return true;
};

File: palindrome.cpp
///PROBLEM #9


#include <string>
using namespace std;

/*
Iterate through first and last index until middle number
*/
class Solution {
public:
    bool isPalindrome(int x) {
        string s = to_string(x);
        int a = 0;
        int b = s.size()-1;
        while (b > a){
            if (s[a] != s[b]){
                return false;
            }
            a++;
            b--;
        }
        return true;
		}
};